examples/common/scripts/vtadmin-down.sh

```diff
@@ -1,9 +1,40 @@
 #!/bin/bash
 
+# Copyright 2023 The Vitess Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 source "$(dirname "${BASH_SOURCE[0]:-$0}")/../env.sh"
 
-echo "Stopping vtadmin-web..."
-kill -9 "$(cat "$VTDATAROOT/tmp/vtadmin-web.pid")"
+function stop_vtadmin() {
+  local name="$1"
+  local file="$2"
+
+  if [[ -z ${1} || -z ${2} ]]; then
```

Comment:
@mattlord : not sure you saw this.

Reply:
If you change this, you can also look at below.
https://github.com/vitessio/vitess/pull/13433#discussion_r1253251606

Reply:
This works as expected:
```
❯ bash -c "source ../common/lib/utils.sh; stop_process"
ERROR: A binary name and PID file must be specified when attempting to shutdown a process
❯ bash -c "source ../common/lib/utils.sh; stop_process foo"
ERROR: A binary name and PID file must be specified when attempting to shutdown a process
❯ bash -c "source ../common/lib/utils.sh; stop_process foo bar"
Skipping stopping foo because the specified PID file (bar) does not exist.
```

And `shellcheck` does not complain about the current usage. -z does not do interpolation to strings like you think. It checks if the variable itself is set.

Reply:
Ok, thanks for checking, til !